### DISCUSSION TITLE
matchUtilities and parseColor issue that breaks Tailwind Intellisense

### DISCUSSION THREAD

**Discussion by luizov** [tailwindlabs/tailwindcss#10699] [Help] [answered]
I am trying to create a dynamic utility using the `matchUtilities` method that takes a color and uses it to add multiple linear gradients and an animation effect to a given element. For this I am using the `parseColor` function from `tailwindcss/lib/util/color` because I need to use rgba in the gradients.
Here is an example: https://play.tailwindcss.com/yCFFigTcxS

However, there is a problem: the OUTPUT terminal throws the following error, coming from Tailwind Intellisense:
`Tailwind CSS: Cannot destructure property 'color' of 'parseColor(...)' as it is null.`
This actually breaks intellisense autocompletion, etc completely. You can see it even in the Tailwind Play example. Any ideas on how to fix the issue?
Thank you.

**Comment by wongjn** [comment, marked as answer]
You could "catch" the `null` return and offer a fallback: https://play.tailwindcss.com/7b8Axi2rtu?file=config

**Reply by luizov** [reply]
Thanks for the reply. Unfortunately, the Tailwind Play example hasn't been updated and I can't see the changes 😅 If I specify a fallback:
`const { color } = parseColor(value) || {color: ['255', '255', '255']}` I think it works.
The OUTPUT now displays `[tailwind.config.js] Building...` permanently though but autocomplete works again.

**Reply by wongjn** [reply]
Updated Tailwind Play link.

**Comment by salvaft** [comment]
I've came accross these utils like `parseColor` and `flattenColorPalette` but haven't found any documentation.
Is there any?
Thanks in advance

**Reply by wongjn** [reply]
Nope, no documentation apart from the source code.
